Documentation/installation/k0s.rst

```diff
@@ -0,0 +1,111 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _k0s_install:
+
+*****************************
+Installation k0s Using k0sctl
+*****************************
+
+This guide walks you through installation of Cilium on `k0s <https://k0sproject.io/>`_,
+an open source, all-inclusive Kubernetes distribution, 
+which is configured with all of the features needed to build a Kubernetes cluster.
+
+Cilium is presently supported on amd64 and arm64 architectures.
```

Comment:
This statement about platform support is generally true, but having this statement here in these docs makes it more likely that we will forget to update this if we change the set of supported platforms. Should amd64/arm64 platform support be mentioned on some more central page, maybe https://docs.cilium.io/en/latest/operations/system_requirements/ ?